portfolio/src/main/webapp/index.html

```diff
@@ -1,50 +1,50 @@
-      </div>
-      <h1><i>Welcome to Olu's Portfolio</h1>
-      <div class="button">
-        <a href="https://www.linkedin.com/in/olua/" target="_blank"></a>
-        <input type="image" alt="LinkedIn" src="images/LI.png">
-        <a href="https://github.com/olu144" target="_blank"></a>
-        <input type="image" alt="GitHub" src="images/GH.png">		
-      </div>
-      <div id="content">
-        <button onclick="getGreetingUsingArrowFunctions()">Click here for a greeting!</button>
-        <div id="greeting-container"></div>
-      </div>
-    </header>      
-  </body> 
-  <footer>
-    &copy; Copyright 2020 Olu's Website
-  </footer> 
-</html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Olu's Portfolio">
+    <meta name="keywords" content="HTML, CSS, JavaScript, Olu, Portfolio, CS">
+    <meta name="author" content="Olu Alamutu">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <script src="script.js"></script>
+    <title>Olu's Portfolio</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
+    <link rel="manifest" href="images/site.webmanifest">
+  </head> 
+  <body>
+    <header>
+      <div class="row">
+        <img src="images/logo.png"
+      </div>
+      <ul class="main-nav">    
+        <li class="active"><a href="" style="color:white"> HOME </a></li>
+        <li><a href="gallery.html" style="color:white"> PHOTOGRAPHY </a> </li>
+	    <li><a href="showcase.html" style="color:white"> PROJECT SHOWCASE </a></li>
+        <li><a href="about.html" style="color:white"> ABOUT </a></li>
+        <li><a href="contact.html" style="color:white"> CONTACT </a></li>
+        <li><a href="stock.html" style="color:white"> STOCK PICKER</a></li>
+        <li><a href="comments.html" style="color:white"> COMMENTS</a></li>
+      </ul>
+      <div class="button">
+        <a href="index.html"></a>
+        <input type="image" alt="Olu" src="images/headshot.png" style="max-width: 600px;max-height: 300px;"> 
+      </div>
+      <h1><i>Welcome to Olu's Portfolio</h1>
+      <div class="button">
+        <a href="https://www.linkedin.com/in/olua/" target="_blank"></a>
+        <input type="image" alt="LinkedIn" src="images/LI.png">
+        <a href="https://github.com/olu144" target="_blank"></a>
+        <input type="image" alt="GitHub" src="images/GH.png">
+      </div>
+      <div id="content">
+        <button onclick="getGreetingUsingArrowFunctions()">Click here for a greeting!</button>
+        <div id="greeting-container"></div>
+      </div>
+    </header>      
+  </body> 
+  <footer>
+    &copy; Copyright 2020 Olu's Website
+  </footer> 
```

Comment:
nit: It's a little unintuitive, but let's put `<footer>` within the `<body>` tag.  Typically, anything that should be displayed on the HTML document should be within `<body>`.